gulp/sass.js

```diff
@@ -96,7 +96,7 @@ module.exports = (gulp, plugins, blueprint) => {
             // package the variables list for consumption -- no imports or functions
             .pipe(plugins.stripCssComments())
             .pipe(plugins.replace(/\n{3,}/g, "\n\n"))
-            .pipe(plugins.replace(/(@import|\/\/).*\n+/g, ""))
+            .pipe(plugins.replace(/(@import|\/\/).*\r\n+/g, ""))
```

Comment:
this change doesn't work on Mac now! can you try change it to `[\r\n]+`? that says "at least one `\r` _or_ `\n`"